source/frontend/src/features/mapSideBar/property/MotiInventoryHeader.tsx

```diff
@@ -81,10 +81,18 @@ export const MotiInventoryHeader: React.FunctionComponent<IMotiInventoryHeaderPr
       <LoadingBackdrop show={isLoading} parentScreen={true} />
       <Row className="no-gutters">
         <Col xs="7">
-          <HeaderField label="Civic Address:" labelWidth={'3'} contentWidth="9">
+          <Row className="justify-content-start">
+            <HeaderLabelCol label="PID #:" labelWidth="3" />
+            <HeaderContentCol>{pid}</HeaderContentCol>
+            <HeaderLabelCol label="PIN #:" />
+            <HeaderContentCol>
+              <span className="pl-2">{pin}</span>
```

Comment:
what does the wrapping span provide?

Reply:
the span is there so we can space the PIN value a but from the pin label. I cannot add a classname to HeaderContentCol because it is already a styled components with its own styles (and it is re-used on other places)